src/WebUI/dotnet/WebPortal/Controllers/HomeController.cs

```diff
@@ -91,7 +91,11 @@ private async Task<bool> AddUser(UserEntry userEntry, string clusterName)
 
             if (userEntry.isAuthorized == "true")
             {
-                var url = clusterInfo.Restapi + "/AddUser?userName=" + HttpContext.Session.GetString("Email") + "&userId=" + userEntry.uid;
+                var url = clusterInfo.Restapi + "/AddUser?userName=" + HttpContext.Session.GetString("Email")
+                    + "&userId=" + userEntry.uid
+                    + "&uid=" + userEntry.uid
+                    + "&gid=" + userEntry.gid
+                    + "&groups=" + JsonConvert.SerializeObject(groups);

```

Comment:
This may be an issue when groups are too long. Let's keep it right now and revisit it later.